### PR TITLE
Add Autopilot API

### PR DESCRIPTION
**Pull request by thierryung**
`\Delighted\AutopilotConfiguration::retrieve($platform)`
https://app.delighted.com/docs/api/getting-autopilot-configuration

`\Delighted\AutopilotMembership\Email::list()`
https://app.delighted.com/docs/api/listing-people-in-autopilot

`\Delighted\AutopilotMembership\Email::create($person)`
https://app.delighted.com/docs/api/adding-people-to-autopilot

`\Delighted\AutopilotMembership\Email::delete($personId)`
https://app.delighted.com/docs/api/removing-people-from-autopilot

### CODE REVIEW

File: lib/Delighted/AutopilotConfiguration.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+{
+    protected static $path = 'autopilot';
+
+    public static function retrieve($platform, Client $client = null)
+    {
+        if (is_null($client)) {
+            $client = Client::getInstance();
+        }
+
+        $response = $client->get(self::$path . '/' . urlencode($platform));
```
Comment (kindjar) [inline lib/Delighted/AutopilotConfiguration.php:15]: Do we need a `validatePlatform` sort of thing in here, like you've put in `AutopilotMembership`?

**Review by kindjar — APPROVED** [review]
Looks pretty good to me. Thanks!

File: lib/Delighted/AutopilotConfiguration.php (hunk 1 above)
Comment (thierryung) [reply lib/Delighted/AutopilotConfiguration.php:15]: We could, but was thinking we could rely on the backend to validate this, which has the slight advantage of needing less configuration in the client code. I also removed the `AutopilotMembership.validatePlatform()` function to favor dedicated classes to mirror the other clients.

```javascript
// GET 'https://api.delighted.com/v1/autopilot/emailz'
{
    "status": 404,
    "message": "Autopilot for emailz does not exist."
}
```

Comment (kindjar) [reply lib/Delighted/AutopilotConfiguration.php:15]: Cool, I was just noting the difference in validation between the two, but it sounds like they are now the same. 😄

**Pull request merged**